addon/globalPlugins/nao.py

```diff
@@ -53,6 +67,11 @@ def script_doRecognizeScreenshotObject(self, gesture):
 			return
 		self.recogUiEnhance.recognizeScreenshotObject()
 
+	@script(
+		# Translators: Message presented in input help mode.
+		description=_("NAO: recognize any sorts of images and pdf from file system."),
```

Comment:
Same remark as above regarding "NAO".

Here is also an optional rephrasing proposal:

```suggestion
		description=_("Recognize the content of the selected image or PDF file."),
```